lib/Serialization/ModuleFormat.h

```diff
@@ -109,6 +109,8 @@ class DeclContextID {
     return rawValue != 0;
   }
 
+  operator uint64_t() const { return getOpaqueValue(); }
```

Comment:
Where is this useful?

Reply:
Thanks for the catch, this was a required by the serialization machinery when I was attempting to serialize a `DeclContext *`. I'll remove it for clarity (though it might be helpful to whoever tries to serialize a `DeclContext *` next as debugging the template errors that resulted from not having this conversion was a bit tedious).